### DISCUSSION TITLE
User Installed Context menus

### DISCUSSION THREAD

**Discussion by ghost** [Pycord-Development/pycord#2483] [Q&A] [answered]
I'm quite new to this but how do I make a User Install Context Menu (In the menu when you right-click on a message)?? (Also please provide some example code that I can go off of.)

**Comment by MattyTheHacker** [comment, marked as answer]
Do you actually mean a user context (right click the user name) or a message context (right click on a message)? The first will give you a user object to work with and the other will give you a message object. The relevant documentation is here: https://guide.pycord.dev/interactions/application-commands/context-menus
